### ISSUE TITLE
laravel 11.x compatibility

### ISSUE THREAD

**Issue by 8ctopus** [illuminatech/url-trailing-slash#14]
Hello! Thank you for a great package, I have been using over the years.
Trying to upgrade to laravel 11, I get the following composer conflict

```sh
$ composer update
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install illuminatech/url-trailing-slash 1.1.3 (conflict analysis result)
    - Conclusion: don't install illuminatech/url-trailing-slash 1.1.5 (conflict analysis result)
    - Conclusion: don't install illuminatech/url-trailing-slash 1.1.7 (conflict analysis result)
    - Conclusion: don't install illuminatech/url-trailing-slash 1.1.8 (conflict analysis result)
    - Conclusion: don't install illuminatech/url-trailing-slash 1.1.10 (conflict analysis result)
    - illuminate/routing[v6.0.0, ..., v6.19.1] require php ^7.2 -> your php version (8.3; overridden via config.platform, same as actual) does not satisfy that requirement.
    - Only one of these can be installed: illuminate/routing[v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.4], laravel/framework[v11.0.0, ..., v11.0.8]. laravel/framework replaces illuminate/routing and thus cannot coexist with it.
    - illuminatech/url-trailing-slash[1.1.0, ..., 1.1.2] require illuminate/routing ^6.0 -> satisfiable by illuminate/routing[v6.0.0, ..., v6.20.44].
    - Root composer.json requires illuminatech/url-trailing-slash ^1.1 -> satisfiable by illuminatech/url-trailing-slash[1.1.0, ..., 1.1.10].
    - Root composer.json requires laravel/framework ^11.0 -> satisfiable by laravel/framework[v11.0.0, ..., v11.0.8].
```

The automatic pull request https://github.com/illuminatech/url-trailing-slash/pull/13/commits fixes the issue.

**Comment by klimov-paul** [comment]
Resolved by commit 7bbc51cee44e275e3f6ca112649fb4a7d64a36bf

**Comment by 8ctopus** [comment]
Thank you Paul!
